optimum/bettertransformer/models/__init__.py

```diff
@@ -100,6 +112,48 @@ def supports(model_type: str) -> bool:
         """
         return model_type in BetterTransformerManager.MODEL_MAPPING
 
+    @staticmethod
+    def requires_nested_tensor(model_type: str) -> bool:
+        """
+        Returns True if the BetterTransformer implementation for a given architecture uses nested tensors, False otherwise.
+
+        Args:
+            model_type (`str`):
+                The model type to check.
+        """
+        if model_type in ["codegen", "gpt2", "gptj", "gpt_neo", "gpt_neox", "opt"]:
```

Comment:
nit: this is all the decoders no? can we just add something like `is_decoder`? feel free to ignore though this is a nitpicking

Reply:
I won't do it now, but yes, the three added methods are very similar, that should be done!